core/src/main/java/software/amazon/awssdk/retry/RetryPolicy.java

```diff
@@ -128,6 +129,7 @@ public int getMaxErrorRetry() {
      * @return Whether this retry policy should honor the max error retry set in
      *         ClientConfiguration
      */
+    @ReviewBeforeRelease("This isn't used.")
```

Comment:
May be used by external customers.

Reply:
It's worthless if we don't use it ourselves, right?

Reply:
Are we setting this value anywhere that the customer could then use this to determine the current value?